### ISSUE TITLE
Probably millis() integer overflow problem

### ISSUE THREAD

**Issue by DanielAndrascik** [philbowles/H4#7]
Hi. After 49 days, all tasks will stop working. I only use a few "every" tasks but after 49 days all are dead. My UDP server in a separate thread is still running, that means ESP32 is alive. I guess it is millis() integer overflow problem.

**Comment by HamzaHajeir** [comment]
I don't think so, even that overflows, the difference is back to zero since it's unsigned.

**Comment by HamzaHajeir** [comment]
Hi @DanielAndrascik, you're correct, and I've just proposed a solution, you can take a look here:
https://github.com/HamzaHajeir/H4/commit/093b019e36127a5c245cb795cf89f298328681fd

**Comment by HamzaHajeir** [comment]
To validate the issue, here's a MCVE:
```cpp
#include <H4.h>
H4 h4(115200);

void testTimer(){
    Serial.printf("T=%lu testTimer()\n", millis());
    h4.once(UINT32_MAX, testTimer);
}
void h4setup(){
    delay(1); // ensure at least one millisecond pass
    testTimer();
}
```

**Comment by HamzaHajeir** [comment]
Solved in my fork.
